activerecord/lib/active_record/relation/batches.rb

```diff
@@ -28,10 +28,11 @@ module Batches
     # ==== Options
     # * <tt>:batch_size</tt> - Specifies the size of the batch. Default to 1000.
     # * <tt>:start</tt> - Specifies the primary key value to start from.
+    # * <tt>:end_at</tt> - Specifies the ending point for the batch processing.
```

Comment:
:+1: on removing the ambiguity.

No, we haven't discussed a range option. Off the top of my head that sounds less clear re. inclusive/exclusive.